app/build.gradle

```diff
@@ -28,12 +28,12 @@ android {
 }
 
 dependencies {
-    implementation "io.didomi.sdk:android:1.86.3"
+    implementation "io.didomi.sdk:android:1.87.0"
 
     implementation "androidx.appcompat:appcompat:1.6.1"
     //noinspection GradleDependency - Higher version will require kotlin
```

Comment:
you can remove this comment, gradle will suggest you a new version

Reply:
True, I updated the dependency: https://github.com/didomi/sample-app-android-java/pull/3/commits/c8d689f710f1125a22698cc3378942d008c7ad72